docs/source/classroom/class_use.md

```diff
@@ -10,23 +10,23 @@ The following Frequently Asked Questions section should help answer most of your
 There are several ways you can use the cluster in your classroom. Your class can use the cluster to access specific software packages and working environments and learn how to utilize high-performance computing (HPC) resources for large and complex data processing, such as machine learning; AI and molecular simulations; and more.
 
 ## How do I get my class access to the cluster?
-You fill out an [HPC Classroom Use Request] ticket. The form asks you to submit a list of your student’s names and emails. We will create accounts on the cluster for them. Follow the steps in the article [How to Download a List of Student Email Addresses from Canvas] to get a list of your students’ emails. If your class roster is incomplete, you can attach a preliminary list to the ticket and follow up with us on the same ticket with an updated roster when the add/drop period is over.
+You fill out an [HPC Classroom Use Request] ticket. While filling out this form, please provide the course number and section, and we will pull the students names directly from canvas. We will create accounts on the cluster for them as well as any TA's or co-instructors. We will also make a directory tree under /courses for the class. If there are any changes in enrollment please let us know so we can update your course. There is no need to send us a list of students names as we pull them directly from canvas.
```

Comment:
Not sure if we should say pick from the dropdown? It is minor but there is a free text section and wouldn't want professors thinking they need to add that information in that box.